lib/fastlane/plugin/wpmreleasetoolkit/actions/common/buildkite_trigger_build_action.rb

```diff
@@ -0,0 +1,90 @@
+module Fastlane
+  module Actions
+    class BuildkiteTriggerBuildAction < Action
+      def self.run(params)
+        require 'buildkit'
+
+        UI.message "Triggering build on branch #{params[:branch]} at #{params[:commit]}"
+
+        pipeline_name = {
+          PIPELINE: params[:pipeline_file]
+        }
+
+        client = Buildkit.new(token: params[:buildkite_token])
+        response = client.create_build(
+          params[:buildkite_organization],
+          params[:buildkite_pipeline],
+          {
+            branch: params[:branch],
+            commit: params[:commit],
+            env: params[:environment].merge(pipeline_name)
+          }
+        )
+
+        response.state == 'scheduled' ? UI.message('Done!') : UI.user_error!("Failed to start job\nError: [#{response}]")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Triggers a job on Buildkite'
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :buildkite_token,
+            env_name: 'BUILDKITE_ACCESS_TOKEN',
+            description: 'Buildkite Personal Access Token',
+            type: String
```

Comment:
TIL about [the `sensitive` option](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L57-L59), so might as well use it (and probably worth adding to our other existing actions at some point too, but that's for a separate PR)
```suggestion
            type: String,
            sensitive: true
```

Reply:
TIL!